app/services/return-versions/setup/check/submit-check.service.js

```diff
@@ -29,6 +30,12 @@ async function go(sessionId, userId) {
 
   await PersistReturnVersionService.go(returnVersionData)
 
+  const dateOffset = 24 * 60 * 60 * 1000
```

Comment:
If this is whole day as an int it could be worded better or a comment added ? 

`oneDayInMilliseconds` - if i have understood it right.